html/components/award.stories.js

```diff
@@ -54,10 +54,9 @@ export const defaultStory = () => {
       class="sprk-o-Stack__item"
       data-sprk-toggle="container"
     >
-      <a
-        class="sprk-b-TypeBodyThree sprk-b-Link sprk-b-Link--simple sprk-b-Link--has-icon"
+      <button
+        class="sprk-b-TypeBodyThree sprk-b-Link sprk-b-Link--simple sprk-b-Link--has-icon sprk-u-BareButton sprk-u-TextCrop--none"
```

Comment:
We arent using the BareButton class anymore per our conversation on video chat yesterday.